### DISCUSSION TITLE
Ready to benchmarks?

### DISCUSSION THREAD

**Discussion by iAxel** [clue/framework-x#109] [Q&A] [answered]
Hello everyone!

Since the day of the announcement, I've been following the framework, will there be performance results?

Try to post results [here](https://github.com/the-benchmarker/website)

**Comment by SimonFrings** [comment, marked as answer]
Hey @iAxel

We're going to make new benchmarks sometime in the future, there are already some up on our [website](https://framework-x.org/). You're always invited to run them on your own, we're happy to see how they changed over time.

We're currently working on the ReactPHP HTTP component (and some other ideas) which will improve Framework X, so there will be some changes to performance in the nearer future.

**Reply by iAxel** [reply]
Thank you for Answer!
